apps/admin/src/components/attributes/EditAttribute/EditAttributeTabs/ActionsListTab/ALCCard/ActionContent/Param/Param.tsx

```diff
@@ -74,6 +71,7 @@ function Param({param, actionId, changeParam, setBlockCard, index}: IParamProps)
                         onChange={_onChange}
                         onFocus={_onFocus}
                         onBlur={_onBlur}
+                        rows={7}
```

Comment:
**polish:** magic number?

Reply:
oui magic number totalement, je pourrais éventuellement mettre 
```suggestion
                        rows={enougthRowToSeeDefaultValues}
```

Reply:
on l'enlève